### ISSUE TITLE
This is a cool mod.but GIF-support will be better

### ISSUE THREAD

**Issue by 8923control** [HashiCraft/fabric-projector-mod#5]
mp4 maybe :b

**Comment by nicholasjackson** [comment]
Yes, I agree, we have plans to add video files and animated gifs to projector, did a quick proof on how this would work and 
I am confident we can make this happen. This might take a while as it would be a big feature.
